festung/dbapi.py

```diff
@@ -161,13 +161,9 @@ def execute(self, operation, parameters=None):
         self._description = _generate_description(response['headers'])
 
     def executemany(self, operation, parameters_sequence):
-        rowcount = 0
-        for parameters in parameters_sequence:
-            self.execute(operation, parameters)
-            rowcount += self._rowcount
-            if self.fetchone() is not NO_MORE_ROW:
-                raise ProgrammingError("The statement shall not produce a result.")
-        self._rowcount = rowcount
+        self.execute(operation, parameters_sequence)
```

Comment:
If you use execute it will use `params` in the request instead of `bulk_params`.

Reply:
lol, stupid me.... thanks!